### ISSUE TITLE
Images not loading on medium.com

### ISSUE THREAD

**Issue by Sh1d0w** [brave/browser-laptop#3682]
**Describe the issue you encountered:**

Images are not loading for this arcticle here https://medium.com/engineering-housing/progressing-mobile-web-fac3efb8b454#.mv2rdvjyq

**Expected behavior:**
The images should load normally

- Platform (Win7, 8, 10? macOS? Linux distro?):
All
- Brave Version:
0.11.6

- Steps to reproduce:
    1. Visit https://medium.com/engineering-housing/progressing-mobile-web-fac3efb8b454#.mv2rdvjyq
    2. The images are not loaded

- Screenshot if needed:
<img src="https://s14.postimg.org/je9p0n9qp/Developer_Tools_https_medium_com_service_wor.png"/>
- Any related issues:

#2965

**Comment by jkup** [comment]
Hmm all loading for me. Any chance you're getting errors in your console that might help us figure out what's going on?

**Comment by Sh1d0w** [comment]
@jkup Updated the issue with a screenshot from the console.

**Comment by srirambv** [comment]
@Sh1d0w disabling FP protection loads all images.

**Comment by Sh1d0w** [comment]
@srirambv You are right! Did not pay attention to it. Closing.

**Comment by bbondy** [comment]
It's useful to investigate things like this because we want people to have a good experience with shields up.

**Comment by jkup** [comment]
Yeah we should still investigate.. also IIRC fingerprint protection used to warn that it may break some sites but doesn't now. Is that correct?

**Comment by anisse** [comment]
Not loading for me on mobile, with all protections disabled.

**Comment by Simdk8** [comment]
Also not showing for me with shields down

![screenshot_2017-05-31-22-46-17](https://cloud.githubusercontent.com/assets/13088589/26653364/33eda2b4-4653-11e7-8eeb-568b7565b125.png)

**Comment by Alino** [comment]
images are not loaded for me on many websites.
For example right now I cannot see half of the images on aliexpress here
https://www.aliexpress.com/wholesale?catId=0&initiative_id=SB_20171213151353&SearchText=safe

if I scroll down, in the middle of the web there are missing images. I can refresh and they are still missing. If I turn shields down, all images get loaded.

**Comment by homotopycolimit** [comment]
I too have never been able to see images on medium.com using brave.
Is the fault on medium's side or on Brave's?
